tools/update.sh

```diff
@@ -46,6 +46,12 @@ yarn run install:prod
 # Build assets for client
 yarn run build
 
+# Migrate database schema
+# In case of a failed migration, we do not stop the application because
+# The database will still be working but will not handle new cases.
+# Anyway, the error will be logged
+yarn run migrate
```

Comment:
So we have to run `migrate.js` each time we want to update the app ? What happen if nothing have changed in the database ?

Reply:
It will be launch by the script and it will ask for mysql information (needed to check if the database is up to date).

Then it will apply every script that isn't in the database

Reply:
Have you ever tested it ? :)

Reply:
I tested the script alone, never the full update script, but it worked :smile: